src/runtime/Makefile

```diff
@@ -173,7 +173,7 @@ DEFSHAREDFS_QEMU_VIRTIOFS := virtio-fs
 DEFVIRTIOFSDAEMON := $(VIRTIOFSDBINDIR)/virtiofsd
 # Default DAX mapping cache size in MiB
 DEFVIRTIOFSCACHESIZE := 1024
```

Comment:
Just a query? Does `DEFVIRTIOFSCACHESIZE` needs to be set to 0 with cache = none or it doesn't really matter ?

Reply:
@bpradipt, nice question. In my mind it'd make sense to also have the cache size set to 0, which would then disable DAX by default.

@dagrh, mind to jump in?

Reply:
The 'cache =' and 'virtiofscachesize' are unfortunately two different caches.
The 'cache =' is about metadata caching, and the 'VIRTIOFSCACHESIZE' is the DAX cache.

Reply:
thanks @dagrh

Reply:
Indeed, thanks @dagrh!